### PR TITLE
Signedoff: raghavaggarwal99<ragha170103048@iitg.ac.in>

### PR DESCRIPTION
**Pull request by raghavaggarwal99**
Signed-off-by: raghavaggarwal99 <ragha170103048@iitg.ac.in>

### CODE REVIEW

**Comment by raghavaggarwal99** [comment]
To the issue #519

**Comment by raghavaggarwal99** [comment]
Hi @leecalcote changes are done, tell me if anything is wrong with them.

**Comment by leecalcote** [comment]
@raghavaggarwal99 here's what I see when I checkout the latest changes:

Screen Shot 2019-12-21 at 9 04 53 AM

File: docs/assets/css/main.css [hunk 1]
```diff
@@ -18602,3 +18608,70 @@ footer {
...
+ 
+  
+}
+
+.footer-icons:hover .footer-icons-text {
+    visibility: visible;
+}
+
+  
+.social__link {
```
Comment (leecalcote) [inline docs/assets/css/main.css]: It's ideal if we can reuse the same code from the layer5 site so as to reduce maintenance of two slightly different implementations. Are you able to use this sass instead? --> https://github.com/layer5io/layer5/blob/master/_sass/_hori-social.scss

Comment (raghavaggarwal99) [reply docs/assets/css/main.css]: Screenshot from 2019-12-21 20-44-52
Are you sure you are updating the latest changes?

Comment (raghavaggarwal99) [reply docs/assets/css/main.css]: and for sass part, i have done exactly same, i just changes the sass code into css

Comment (leecalcote) [reply docs/assets/css/main.css]: Understood. Can we leave as sass and literally reuse the same code? This type of reuse practice improves sustainability and quality.

Comment (raghavaggarwal99) [reply docs/assets/css/main.css]: okay will do.

Comment (leecalcote) [reply docs/assets/css/main.css]: With respect to building the latest changes, yes, when I check out the branch (PR) and run `git log -1`, I see:

```
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
commit ace454a3d315f6786c040abea41d6d9111cb84b5 (HEAD -> pull/527)
Author: raghavaggarwal99 <ragha170103048@iitg.ac.in>
Date:   Sat Dec 21 17:06:59 2019 +0530

    Signedoff: raghavaggarwal99<ragha170103048@iitg.ac.in>

    Signed-off-by: raghavaggarwal99 <ragha170103048@iitg.ac.in>

    Signed-off-by: raghavaggarwal99 <ragha170103048@iitg.ac.in>
```
This is the same commit ID as is listed as the latest ID above. I see the same behavior in both Safari and Chrome.

Comment (leecalcote) [reply docs/assets/css/main.css]: > okay will do.

thank you, sir. 👍

**Comment by leecalcote** [comment]
@raghavaggarwal99 I'm still seeing this issue - https://github.com/layer5io/meshery/pull/527#issuecomment-568187818

Will you investigate?

**Comment by raghavaggarwal99** [comment]
Cool, I was just doing it

**Pull request merged**